data/remnant missions.txt

```diff
@@ -1,497 +1,1358 @@
-# Copyright (c) 2017 by Michael Zahniser
```

Comment:
Looks like your editor mangled line endings as well. You're gonna need to fix that.

Reply:
Hmmm, I'll take a look at that. I'm using NP++. Thanks for pointing it out.

Reply:
Just checking, and has been resolved.